app/models/like.rb

```diff
@@ -0,0 +1,6 @@
+class Like < ActiveRecord::Base
```

Comment:
I'm not sure what a 'duty' means, maybe even just a comment saying what exactly `duty` means could help a lot